### PR TITLE
Include RIDs

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: testapps/SimpleApp/SimpleApp.csproj [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
-    <RuntimeIdentifier Condition="!$(TargetFramework.StartsWith('netcoreapp'))">win7-x64</RuntimeIdentifier>
+    <RuntimeIdentifiers Condition="!$(TargetFramework.StartsWith('netcoreapp'))">win7-x86;win7-x64;linux-x64;osx-x64</RuntimeIdentifiers>
```
Comment (Tratcher) [inline testapps/SimpleApp/SimpleApp.csproj]: Do we know why the StartsWith is there? Do tests pass with it?

File: testapps/SimpleApp/SimpleApp.csproj [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
-    <RuntimeIdentifier Condition="!$(TargetFramework.StartsWith('netcoreapp'))">win7-x64</RuntimeIdentifier>
```
Comment (pranavkm) [inline testapps/SimpleApp/SimpleApp.csproj]: What's this for? These tests specifically run only for win7-x64.

File: test/FunctionalTests/PublishForRidTest_CoreCLR.cs [hunk 3]
```diff
@@ -1,72 +0,0 @@
...
-{
-    public class PublishWithRidTest_CoreCLR : LoggedTest
-    {
-        public PublishWithRidTest_CoreCLR(ITestOutputHelper output)
-            : base(output)
-        {
-        }
-
-        [Fact]
-        public void CrossPublishingWorks()
```
Comment (Tratcher) [inline test/FunctionalTests/PublishForRidTest_CoreCLR.cs]: ? killed the whole test?

Comment (pranavkm) [reply test/FunctionalTests/PublishForRidTest_CoreCLR.cs]: We discussed this - the test is some what marginal value and sounds like making it work would be painful.

**Comment by ryanbrandenburg** [comment]
CLosing while @pranavkm does his investigation.

**Pull request closed without merging**